source/documentation/getting_started/quick_start.md

```diff
@@ -1,7 +1,7 @@
 ## Quick start guide
 
-1. [Find the base URL for the register(s) you want to use](https://registers.cloudapps.digital/registers)
-2. [Find the endpoints you want](#apireference).
+1. [Find the base URL for the register(s) you want to use](https://registers.cloudapps.digital/registers). For example, for the 'Local authority eng' register: `https://local-authority-eng.register.gov.uk/`.
+2. [Find the endpoints you want](#apireference). For example, `/record/BIR` in `https://local-authority-eng.register.gov.uk/record/BIR/`.
```

Comment:
no need for the trailing `/` on the URI

Reply:
Thanks @gidsg — I'll remove all trailing `/`s here and other PRs